merlin/models/tf/transforms/features.py

```diff
@@ -741,10 +741,7 @@ def call(
         self, inputs: TabularData, targets=None, testing=False, **kwargs
     ) -> "PredictionOutput":
         if testing:
```

Comment:
Are `training` and `testing` flags provided when a transform block is used with the Loader?
Or do we also want to support the usage of `ToTarget()` inside the model?

Reply:
Thanks, I now think it's better to drop this parameter entirely because I can't think of an exact scenario where this would be used. Maybe we can add it back if needed.